osu.Game/Online/Leaderboards/Leaderboard.cs

```diff
@@ -87,6 +88,21 @@ public IEnumerable<TScoreInfo> Scores
             }
         }
 
+        public TScoreInfo TopScore
+        {
+            get => topScoreContainer.Score.Value;
+            set
+            {
+                if (value == null)
+                    topScoreContainer.Hide();
+                else
+                {
+                    topScoreContainer.Show();
+                    topScoreContainer.Score.Value = value;
```

Comment:
Although this code was just moved from its original place, it's a bit weird to me how a `null` top score won't persist to the bindable, contrary to everything else... `UserTopScoreContainer` looks to be able to handle nulls ok.